frontend/benefit/handler/src/components/applicationReview/ApplicationReview.sc.ts

```diff
@@ -15,10 +15,32 @@ type TabButtonProps = {
   active?: boolean;
 };
 
-export const $ApplicationReview = styled.div`
+type ApplicationReviewProps = {
```

Comment:
I would only add extra props for styled components if they directly affected the resulting style. That's not the case currently and this `editDisabled` is not used at all; in `ApplicationReview` itself, all uses of `editDisabled` come directly from the constant received from `useApplicationReview`.

Reply:
Yeah, I had a &:before element there previously which used the prop on for `display`. Will remove this.